web3/_utils/method_formatters.py

```diff
@@ -1190,7 +1187,7 @@ def filter_wrapper(
 
 @to_tuple
 def apply_module_to_formatters(
-    formatters: Tuple[Callable[..., TReturn]],
```

Comment:
Curious about the reasoning behind this change?

Reply:
This change just indicates to mypy that the function will work with any iterable of callables.

The original hint indicates that the function must take a tuple input, so while it didn't cause any type errors internally here it can cause unneeded type errors for downstream users.